File: restaruant/lib/Models/Producto.dart
class Producto {
  int idProducto;
  String nombre;
  String descricion;
  String precio;
  int idTopping;

  Producto({
    required this.idProducto,
    required this.nombre,
    required this.descricion,
    required this.precio,
    required this.idTopping,
  });

  factory Producto.fromJson(Map<String, dynamic> json) {
    return Producto(
      idProducto: json['idProducto'],
      nombre: json['nombre'],
      descricion: json['descricion'],
      precio: json['precio'],
      idTopping: json['idTopping'],
    );
  }
}

File: RestaurantAPIS/RestaurantAPIS/Models/Producto.cs
namespace RestaurantAPIS.Models
{
    public class Producto
    {
        public int idProducto { get; set; }
        public string nombre { get; set; }
        public string descricion { get; set;}
        public string precio { get; set;}
        public int idTopping { get; set; }
    }
}
